### DISCUSSION TITLE
Water and Sewerage Module specifications

### DISCUSSION THREAD

**Discussion by KrisP-Vassar** [egovernments/Digit-Core#47] [Water-Sanitation]
Dear Digit team,
 Please share/upload the specifications for Water and Sewerage Module under documentation site at https://standards.digit.org/.  
Thanks
Krishna P
Vassar Labs

**Comment by subhashini-egov** [comment]
@KrisP-Vassar - https://standards.digit.org/water-and-sewerage-module-specs/sewerage-data-model-specs 
The specs are available in the above link.
